eisbuk-admin/src/components/atoms/DateSwitcher/DateSwitcher.stories.tsx

```diff
@@ -20,6 +20,7 @@ export const Default = (): JSX.Element => (
       open={true}
       anchorEl={null}
       onClose={() => {}}
+      jump={"week"}
```

Comment:
A small thing: this could just be `jump="week"` - if you have a simple string, there's no need for curly braces...there was an eslint rule for this, but we obviously don't implement that :thinking: